lib/packages_groups_xml_creator.py

```diff
@@ -67,7 +99,39 @@ def create_group_xml(group_name, group_pkgs):
     return grp
 
 
-def create_comps_xml(packages_groups):
+def create_environment_xml(environment_name, environment_groups):
+    """
+    Construct an XML object representing an 'environment' element.
+    Those elements will appear as options during the installation.
+
+    Args:
+        environment_name (str): name of an environment of package groups
```

Comment:
If you explain what an  environment is above, IMHO you could simply use "packages environment" instead of "environment of packages groups" here.